indico/modules/categories/templates/display/base.html

```diff
@@ -22,6 +22,11 @@ <h1 class="category-title {{ category_title_classes }}">
             <div class="group">
                 {{ create_event_button(category, classes="highlight", text=_("Create event"), with_tooltip=false) }}
             </div>
+            <div class="group">
+                <div id="navigate-button" class="i-button icon-compass2">
+                    {% trans %}Navigate categories{% endtrans %}
```

Comment:
Just wondering.. do we need the text at all?

![image](https://user-images.githubusercontent.com/179599/77539043-ff950380-6ea0-11ea-8540-b848742a21a3.png)

or at least shorter text?

![image](https://user-images.githubusercontent.com/179599/77539079-1176a680-6ea1-11ea-8ad8-659d180df069.png)


or maybe a button in the bar next to the parent one?

![image](https://user-images.githubusercontent.com/179599/77539342-7e8a3c00-6ea1-11ea-8e46-33913935b94d.png)
![image](https://user-images.githubusercontent.com/179599/77539354-84801d00-6ea1-11ea-8dc1-6c5f1226206d.png)

(i prefer the last option tbh)

```diff
diff --git a/indico/modules/categories/templates/display/base.html b/indico/modules/categories/templates/display/base.html
index 5d92c13a8b..120f61148b 100644
--- a/indico/modules/categories/templates/display/base.html
+++ b/indico/modules/categories/templates/display/base.html
@@ -22,17 +22,14 @@
             <div class="group">
                 {{ create_event_button(category, classes="highlight", text=_("Create event"), with_tooltip=false) }}
             </div>
-            <div class="group">
-                <div id="navigate-button" class="i-button icon-compass2">
-                    {% trans %}Navigate categories{% endtrans %}
-                </div>
-            </div>
             <div class="group">
                 {% if not category.is_root %}
                     <a class="i-button icon-arrow-up" href="{{ category.parent.url }}">
                         {% trans %}Parent category{% endtrans %}
                     </a>
                 {% endif %}
+                <button id="navigate-button" class="i-button icon-only icon-compass2"
+                        title="{% trans %}Navigate categories{% endtrans %}"></button>
                 <a href="#"
                    class="i-button icon-calendar arrow js-export-ical"
                    title="{% trans %}Export to scheduling tool{% endtrans %}"
```

Reply:
This is a matter of ux affordances, discoverability and how much attention we want to draw towards this feature. 

While the icon is nonintrusive, it doesn't communicate that it's now possible to navigate categories. The button with the *Navigate* text does a better job at drawing attention, but it doesn't communicate what exactly can be navigated. 

The text could be avoided if the button was located somewhere that is intimately related with category navigation (like right above list of subcategories or next to breadcrumbs), but since it's not, I prefer to keep the action explicit.

At the same time, I don't have a particularly strong feeling about this. I would, at the very least, leave the button with the *Navigate* text on it. Let me know what you think.

Reply:
Shorter text plus more verbose tooltip? (i'd still move it in the existing group though)

Reply:
While I feel the icon without text looks the best, I feel like that will get easily overlooked. Keeping the text short (`Navigate`) sounds like the best compromise. And I think a more verbose tooltip would be good!

Reply:
I agree with @Leats even though I don't see a huge difference in having just the icon.

Reply:
let's go for that option then.. seems like a good middle ground between a large button and just the icon

Reply:
- Button text changed to `Navigate`.
- Tooltip `Navigate categories` added.

Regarding the grouping, I think it makes more sense to keep the button in a separate group.
|a|b|
|:-|:-|
|![Screen Shot 2020-03-25 at 17 14 43](https://user-images.githubusercontent.com/716307/77559631-b05ccc00-6ebc-11ea-8685-8f82baddcfb3.png)|![Screen Shot 2020-03-25 at 17 15 04](https://user-images.githubusercontent.com/716307/77559641-b5218000-6ebc-11ea-816b-735e2dfb502e.png)| 

In `a`, each group has a clearly different semantic: 1) create stuff, 2) navigate *all* categories, 3) do stuff in *this* category. In `b`, the semantics get mixed up.